### PR TITLE
T5685: Keepalived VRRP prefix is not necessary for the virtual address

### PR DESCRIPTION
**Pull request by natali-rs1985**
## Change Summary

Keepalived VRRP prefix is not necessary for the virtual address

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Migration from an old Vyatta component to vyos-1x, please link to related PR inside obsoleted component
- [ ] Other (please describe):

## Related Task(s)

* https://vyos.dev/T5685

## Component(s) name

high-availability_vrrp

## How to test

```
configure
set high-availability vrrp group GRP01 interface 'eth1'
set high-availability vrrp group GRP01 vrid '254'
set high-availability vrrp group GRP01 address 192.0.2.112
commit
```

## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-1x/blob/current/CONTRIBUTING.md) document
- [x] I have linked this PR to one or more Phabricator Task(s)
- [x] I have run the components [**SMOKETESTS**](https://github.com/vyos/vyos-1x/tree/current/smoketest/scripts/cli) if applicable
- [x] My commit headlines contain a valid Task id
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: interface-definitions/high-availability.xml.in [hunk 1]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0"?>
```
Comment (sever-sever) [inline interface-definitions/high-availability.xml.in]: Why was this line deleted?

Comment (natali-rs1985) [reply interface-definitions/high-availability.xml.in]: it was mistakenly removed. fixed now

**Comment by sever-sever** [comment]
@MergifyIo backport sagitta
